pandas/errors/__init__.py

```diff
@@ -32,6 +32,8 @@ class UnsortedIndexError(KeyError):
 class ParserError(ValueError):
     """
     Exception that is raised by an error encountered in `pd.read_csv`.
+
+    e.g. HTML Parsing and  malformed input csv files will raise this error.
```

Comment:
this is partially redudant

Reply:
Ok, I just looked at various flows that could raise this error. I will update it to just have the HTML parsing.